hooks/environment

```diff
@@ -36,7 +41,17 @@ if [[ "${BUILDKITE_AGENT_META_DATA_PLATFORM}" != "macos" ]]; then
   exit 4
 fi
 
-if [[ "${BUILDKITE_AGENT_META_DATA_HOSTNAME}" != "${permitted_signing_host}" ]]; then
-  echo "Host ${BUILDKITE_AGENT_META_DATA_HOSTNAME} is not permitted to do code-signing."
-  exit 5
+#if [[ "${BUILDKITE_AGENT_META_DATA_HOSTNAME}" != "${permitted_signing_host}" ]]; then
+#  echo "Host ${BUILDKITE_AGENT_META_DATA_HOSTNAME} is not permitted to do code-signing."
```

Comment:
Maybe make this message a plugin variable? I can see it wanting to be customisable so different orgs can provide different actionable instructions to help their users.

Reply:
Nonblocking.